etc/keepass.profile

```diff
@@ -1,6 +1,9 @@
 # keepass password manager profile
-noblacklist ${HOME}/.config/keepass
 noblacklist ${HOME}/.keepass
+noblacklist ${HOME}/.config/keepass
+noblacklist ${HOME}/.config/KeePass
+noblacklist ${HOME}/*.kdbx
+noblacklist ${HOME}/*.kdb
```

Comment:
I don't think we should make .kdbx files available to keepass, or .kdb files available to keepassx.